src/ontology/cl-edit.owl

```diff
@@ -5721,9 +5722,13 @@ SubClassOf(obo:CL_0000238 obo:CL_0000240)
 
 # Class: obo:CL_0000239 (brush border epithelial cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0000239 "An epithelial cell characterized by the presence of a brush border on its apical surface, which increases the surface area for absorption. These cells are predominantly found lining the small intestine and the proximal tubules of the kidneys.")
```

Comment:
> These cells are predominantly found lining the small intestine and the proximal tubules of the kidneys.

Since not all cells are found in this epitheliums, consider moving this sentence to the comments.